### PR TITLE
* Added `movementRatio` prop, this will allow the box to stick to the…

### PR DESCRIPTION
**Pull request by martinmcneela**
This PR doesn't have an existing issue (that I could see).

**The problem:**
When using flexbox the user would see a detachment in the box and the cursor that is resizing it.

**The solution:**
I have added `movementRatio` prop that is not required and defaults to '1' so it won't effect existing functionality.
When using flexbox layout (or anytime you want) you can set the prop to '2' to allow the handle being dragged to stick to the cursor.

**Testing:**
I have tested with and without this props and everything is working as expected. I have also added two now stories to illustrate the issue it is solving.

**Notes:**
This is my first ever PR to an open source repo, so I am happy to take any feedback and make the required changes.

### CODE REVIEW

**Comment by martinmcneela** [comment]
This build failed, it seas 'movementRatio' is undefined even though the default prop is set to 1. Any ideas?

**Comment by martinmcneela** [comment]
Great I'll give that a go first chance I get.

**Comment by martinmcneela** [comment]
@bokuweb that fixed it thanks.

File: src/index.js [hunk 1]
```diff
@@ -120,7 +120,8 @@ export type ResizableProps = {
   onResize?: ResizeCallback,
   onResizeStop?: ResizeCallback,
   defaultSize?: Size,
-  scale?: number
+  scale?: number,
+  movementRatio?: number,
```
Comment (bokuweb) [inline src/index.js]: @martinmcneela Sorry for late. I think it is better to rename it to `resizeRatio`.
And could you please add this new props to `README` ?

**Comment by martinmcneela** [comment]
Change request made and updated.

**Comment by bokuweb** [comment]
@martinmcneela Sorry for late. LGTM

**Pull request merged**